x-pack/plugins/canvas/public/lib/get_pretty_shortcut.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const getPrettyShortcut = (shortcut: string): string => {
+  if (!shortcut) {
+    return '';
+  }
+
+  let result = shortcut.toUpperCase();
+  result = result.replace(/command/i, '⌘');
```

Comment:
Is this not platform specific?  Wouldn't Windows machines show these?

Reply:
In the places that I invoke this function, I only pass in the platform specific shortcut for the function to format. I could do an additional check for the OS here, but I don't think it's necessary.